src/retrace/retrace.py

```diff
@@ -1760,7 +1760,14 @@ def prepare_debuginfo(self, vmcore, chroot=None, kernelver=None, crash_cmd=["cra
                               stdin=PIPE, stdout=PIPE, stderr=null)
         else:
             child = Popen(crash_cmd + ["-s", vmcore, vmlinux], stdin=PIPE, stdout=PIPE, stderr=STDOUT)
-        stdout = child.communicate("mod\nquit")[0]
+        try:
+            t = 3600
+            if CONFIG["ProcessCommunicateTimeout"]:
+                t = CONFIG["ProcessCommunicateTimeout"]
+            stdout = child.communicate("mod\nquit", timeout=t)[0]
```

Comment:
`TypeError: __init__() got an unexpected keyword argument 'timeout'`
`timeout` is not in Python2 subprocess.
It is, however, available in package https://pypi.python.org/pypi/subprocess32/

Reply:
Weird. I had not seen this error but I test on RHEL6 mostly. Did this move in another python version?

Reply:
I tried it with python2.7 on Fedora 27, RHEL7 and failed with the error I mentioned above. I also don't see it in documentation.

Have you tried it? And are you sure it does not always raise exception?  Can you check what exception it raises? I think it will be `TypeError`.

Reply:
I don’t submit pull requests that are not tested. I am testing the patch (RHEL6) and I’ve never seen that exception so it must be a difference. The patch works as designed to kill a long running crash but does not affect other short running crash processes. If we need a fix up for compatibility that is fine with me.

Reply:
That is very wierd. What exact version of python do you have?
Can you run this in you python interperter?
```
from subprocess import Popen, STDOUT                                                                            
child = Popen(["sleep", "10000m"], stderr=STDOUT)                               
output = child.communicate("sleep", timeout=2)  
```

Reply:
$ python
Python 2.6.6 (r266:84292, Aug  9 2016, 06:11:56) 
[GCC 4.4.7 20120313 (Red Hat 4.4.7-17)] on linux2
Type "help", "copyright", "credits" or "license" for more information.
>>> from subprocess import Popen, STDOUT
>>> child = Popen(["sleep", "1000m"], stderr=STDOUT)
>>> output = child.communicate("sleep", timeout=2)
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/usr/lib64/python2.6/subprocess.py", line 732, in communicate
    stdout, stderr = self._communicate(input, endtime)
  File "/usr/lib64/python2.6/subprocess.py", line 1352, in _communicate
    self.wait(timeout=self._remaining_time(endtime))
  File "/usr/lib64/python2.6/subprocess.py", line 1302, in wait
    raise TimeoutExpired
subprocess.TimeoutExpired